### PR TITLE
[#138] Community nation filter (connects #138)

### PR DESCRIPTION
**Pull request by Rupesh87**

### CODE REVIEW

File: src/app/community/community.component.html [hunk 1]
```diff
@@ -23,32 +26,38 @@ <h2 i18n>Communities List</h2>
...
+        <td>
+          {{community.doc.nationName}}
+        </td>
+        <td>
+          {{community.doc.lastPublicationsSyncDate}}
+        </td>
+        <td>
+          {{community.doc.lastActivitiesSyncDate}}
+        </td>
+        <td i18n>
```
Comment (lmmrssa) [inline src/app/community/community.component.html]: need to check if it works here

File: src/app/community/community.component.html [hunk 2]
```diff
@@ -23,32 +26,38 @@ <h2 i18n>Communities List</h2>
...
+          {{community.doc.registrationRequest}}
+        </td>
+        <td>
+          <button i18n class="btn btn-danger"  (click)="deleteCommunity(community.doc._id,community.doc._rev,i)">Delete</button>
```
Comment (lmmrssa) [inline src/app/community/community.component.html]: lets have new issue to add view button here and create nation detail view.

File: src/app/community/community.component.html [hunk 3]
```diff
@@ -23,32 +26,38 @@ <h2 i18n>Communities List</h2>
...
+          {{community.doc.nationName}}
+        </td>
+        <td>
+          {{community.doc.lastPublicationsSyncDate}}
+        </td>
+        <td>
+          {{community.doc.lastActivitiesSyncDate}}
+        </td>
+        <td i18n>
+          {{community.doc.registrationRequest}}
```
Comment (lmmrssa) [inline src/app/community/community.component.html]: No status data here

**Comment by mappuji** [comment]
Do review with this step please 
https://www.youtube.com/watch?v=SmZgAULwoh4

**Pull request merged**